src/main/java/liquibase/ext/clickhouse/sqlgenerator/UpdateChangeSetChecksumClickHouse.java

```diff
@@ -45,16 +48,30 @@ public boolean supports(UpdateChangeSetChecksumStatement statement, Database dat
       UpdateChangeSetChecksumStatement statement,
       Database database,
       SqlGeneratorChain sqlGeneratorChain) {
+    Map<String, String> properties = ParamsLoader.getLiquibaseClickhouseProperties();
+    boolean isReplicated = properties != null;
+
     ChangeSet changeSet = statement.getChangeSet();
     String updateChecksumQuery =
-        String.format(
-            "ALTER TABLE %s.%s UPDATE MD5SUM = '%s' WHERE ID = '%s' AND AUTHOR = '%s' AND FILENAME = '%s'",
-            database.getDefaultSchemaName(),
-            database.getDatabaseChangeLogTableName(),
-            changeSet.generateCheckSum().toString(),
-            changeSet.getId(),
-            changeSet.getAuthor(),
-            changeSet.getFilePath());
+        !isReplicated
+            ? String.format(
```

Comment:
I think we could format the common denominator and isolate the parameter which is added by isReplicated.